app/scripts/directives/footer-version.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+/* global isCoreVersionCompatible:false */
+
+// Common directive for Footer Version
+angular.module('openhimConsoleApp')
+  .directive('footerVersion', function(Api, config) {
+    return {
+      template:'<span ng-class="footerVersionsCompatible ? \'\' : \'version-incompatible\'" ng-if="footerConsoleVersion && footerCoreVersion">' +
+                  '(Console v{{ footerConsoleVersion }}, Core v{{ footerCoreVersion }})' +
```

Comment:
Maybe update the display to show '- (Console <strong>v{{ footerConsoleVersion }}</strong> | Core <strong>v{{ footerCoreVersion }}</strong>)'

Have the version shows in bold and pipe the core and console instead of comma separate it